index.html

```diff
@@ -16,16 +16,170 @@
+      <section>
+      <div class="row">
+        <div class="col-12 col-sm-5">
+          <div id="about">
+            <h2>About</h2>
+            <p>The name is Luke and I'm an aspiring web deveoper and designer, with extensive practical experience in brand strategy and UX/UI; devoted to Functional Programming and Information Architecture. Also a huge fan of semantics and futuristic interfaces.</p>
+            <button type="button" class="btn btn-info">Info</button>
+          </div><!-- about -->
+        </div>
+        <div class="col-12 col-sm-7">
+          <div id="skills">
+            <div class="row">
+              <div class="col-6 front">
+            <h2> Front End Development</h2>
+            <div>
+              <p>HTML5</p>
+              <progress max="100" value="90"></progress>
+          </div>
+          <div>
+              <p>CSS3</p>
+              <progress max="100" value="80"></progress>
+          </div>
+          <div>
+              <p>jQuery</p>
+              <progress max="100" value="60"></progress>
+          </div>
+        </div>
+          <div class="col-6 backend">
+            <h2> Back End Development</h2>
+            <div>
+              <p>Ruby</p>
+              <progress max="100" value="70"></progress>
+          </div>
+          <div>
+              <p>Javascript</p>
+              <progress max="100" value="60"></progress>
+          </div>
+          <div>
+              <p>Node.js</p>
+              <progress max="100" value="40"></progress>
+          </div>
+          </div>
+          </div><!-- inner row -->
+
+          </div><!-- skills -->
+        </div>
+      </div><!-- row -->
+    </section>
+
+        <!-- Button trigger modal -->
+        <div class="secret">
+        <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModal">
+          Secret Message
+        </button>
+
+        <!-- Modal -->
+        <div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h5 class="modal-title" id="exampleModalLabel">Urget Message</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+              <div class="modal-body">
+                This website is NOWHERE FINISHED!
+              </div>
+              <div class="modal-footer">
+                <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+    
+
+    <div class="row footer">
+            <div class="col-12">
+            <p>Copyright Lukasz <span class="highlight">"Velsu"</span> Busk 2018</p>
```

Comment:
You can make this portion has a footer (extending all the way across) so it balances out the navigation on top.